shields.example.env

```diff
@@ -0,0 +1,19 @@
+# Base URL for redirects etc.
+BASE_URL=http://localhost:8080
+# Where your homepage is
+INFOSITE=http://localhost:8080/try.html
+
+# GitHub settings
+GITHUB_URL=https://api.github.com
+# Create your GitHub OAuth application here: https://github.com/settings/developers
+GH_CLIENT_ID=insert_client_id
+GH_CLIENT_SECRET=insert_client_secret
+# Create your Personal Access Token here: https://github.com/settings/tokens
+GH_TOKEN=insert_token
+
+# Server settings
+PORT=80
+BIND_ADDRESS=::
+
+# IP address of your Shields server. You may use your DNS here, but that's less secure
+SHIELDS_IP=insert_shields_ip
```

Comment:
It's used [here](https://github.com/badges/shields/blob/master/lib/github-auth.js#L92) to add GitHub tokens to running Shields servers. I guess an IP address is used over DNS to improve security (protect against DNS hijacking I presume). I'm sure @espadrine might be able to explain this better than me :).